src/backend/api/serializers.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import serializers
+from rest_framework_gis import serializers as gis_serializers
+
+from .models import Neighborhood
+
+
+class NeighborhoodSerializer(gis_serializers.GeoFeatureModelSerializer):
```

Comment:
I initially investigated combining these two into a single serializer and endpoint so that we can only make a single API request, but it quickly became clear that that approach would have required additional surgery on the frontend, so I ended up creating endpoints that mimic the format of the existing data files.